pumps/prometheus.go

```diff
@@ -303,6 +305,23 @@ func (pm *PrometheusMetric) InitVec() error {
 	return nil
 }
 
+// EnsureLabels ensure the data validity and consistency of the metric labels
+func (pm *PrometheusMetric) EnsureLabels() {
```

Comment:
@sredxny please have in mind that if it's found, it should be in the first position of the labels slice. If not, we should move it there.

Reply:
And add some tests over this func 👍

Reply:
@tbuchaillot well, I tested and found out that not necesarily it has to be in the first position. It works even if I put it in the last position

Reply:
This is the output when consuming the /metrics endpoint, so, no matter the position of the label it will work:

```
# TYPE tyk_http_latency histogram
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="1"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="2"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="5"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="7"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="10"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="15"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="20"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="25"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="30"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="40"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="50"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="60"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="70"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="80"} 0
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="90"} 1
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="100"} 2
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="200"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="300"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="400"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="500"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="1000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="2000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="5000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="10000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="30000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="60000"} 3
tyk_http_latency_bucket{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type="",le="+Inf"} 3
tyk_http_latency_sum{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type=""} 374
tyk_http_latency_count{alias="",api_key="GET",api_name="200",method="test-pump",path="/uuid",response_code="total",type=""} 3
```

Reply:
@sredxny not really, looks at the values response_code="total",type="",